packages/gestalt/src/RadioGroup.js

```diff
@@ -50,6 +51,14 @@ function RadioGroup({
   legend,
   legendDisplay = 'visible',
 }: Props): Node {
+  // Consume GlobalEventsHandlerProvider
+  const { onLoad } = useGlobalEventsHandlerContext()?.radioGroupHandlers ?? {
+    onLoad: () => {},
+  };
+  useEffect(() => {
+    onLoad?.();
+  });
```

Comment:
useEffect(() => {
    onRender?.();

  }, [onRender]);

Reply:
https://gestalt.pinterest.systems/web/utilities/globaleventshandlerprovider#Other-handlers

Reply:
DONE

Reply:
Just for clarification, `onLoad` should only be the first time right? or does `onLoad` mean every render. It's different than onMount? 

Does this need an empty dep array at the end of the use effect?